packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
```

Comment:
Nit: Let's use the current flutter_webview version as a starting point. It's kind of weird to have it look like this is a completely new thing.

Reply:
I agree and updated the version to 2.0.13 (same as current webview_flutter package).